src/components/dashboard/main-content/course-enrollments/course-cards/ApprovedCourseCard.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+import { Button } from '@openedx/paragon';
+import { Link } from 'react-router-dom';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import BaseCourseCard from './BaseCourseCard';
+import { COURSE_STATUSES } from '../../../../../constants';
+
+const ApprovedCourseCard = (props) => {
+  const {
+    linkToCourse,
+  } = props;
+  const renderButtons = () => (
+    <Button
+      as={Link}
+      to={linkToCourse}
+      className="btn-xs-block"
+      variant="inverse-brand"
+    >
+      <FormattedMessage
+        id="enterprise.learner-portal.dashboard.courses.assignments.assignment.enroll"
```

Comment:
use camel case standard for id. For example instead of learner-portal use learnerPortal